templates/dashboard/customer/modal/confirm_promote.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+
+<form action="{% block action_url %}#!{% endblock %}" method="post" novalidate>
```

Comment:
Why to use blocks in this template? It's not inherited anywhere.

Reply:
I assumed that with development of our dashboard we eventualy will have more confirmation modals. So i have done to be like base_modal_confirm_delete.html.
Should i delete those blocks tags or add base_modal_confirm template?